### ISSUE TITLE
Check any video or audio content is properly described

### ISSUE THREAD

**Issue by Angelsaur** [GeorgiaHCA/AutSPACEs#48]
### Summary

You’ll need to check that any videos or audio content are clearly described so that users who cannot hear them can still access the information

### Deadline 

20/02/2021

### What needs to be done?

This firstly means checking that videos have captions explaining any sound effects and dialogue. You should also check your sample videos (maybe 2 or 3) to assess the quality of those captions.

Once you’ve done that, you’ll need to check there are transcripts for any audio content you’ve got.

**Comment by gedankenstuecke** [comment]
Made issue #369 to create an accessibility checklist that can be used for the MVP and going forward. Closing this along with other accessibility checklist items for now.
